src/main/resources/templates/db_scripts/data-migration/update_voucherNumver_in_voucher_invoice.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE ${myuniversity}_${mymodule}.vouchers
+SET jsonb = jsonb || jsonb_build_object('voucherNumber', jsonb ->> 'voucherNumber');
+
+UPDATE ${myuniversity}_${mymodule}.invoices AS i
+SET jsonb = jsonb || jsonb_build_object('voucherNumber', v.jsonb ->> 'voucherNumber')
+  FROM ${myuniversity}_${mymodule}.vouchers AS v
+WHERE i.invoiceId = v.invoiceId;
```

Comment:
we also need to update schema.json file to include this fix, next release is 5.9.0

Reply:
sure, I will update the schema.json too, thank you for letting me know : )